card_generator/tasks/cards.py

```diff
@@ -12,6 +13,38 @@
 logger = logging.getLogger(__name__)
 
 
+class OPENSPPCeleryTask(Task):
+    max_retries = 3
```

Comment:
@renceInbox this variable seems to be unused. If it is not unused, this setting should be provided through an environment variable.